lib/lighthouse/letters_generator/service.rb

```diff
@@ -53,31 +63,23 @@ def transform_military_services(services_info)
       end
 
       def transform_benefit_information(info)
-        transform_targets = {
-          awardEffectiveDateTime: :awardEffectiveDate,
-          chapter35Eligibility: :hasChapter35Eligibility,
-          nonServiceConnectedPension: :hasNonServiceConnectedPension,
-          serviceConnectedDisabilities: :hasServiceConnectedDisabilities,
-          adaptedHousing: :hasAdaptedHousing,
-          individualUnemployabilityGranted: :hasIndividualUnemployabilityGranted,
-          specialMonthlyCompensation: :hasSpecialMonthlyCompensation
-        }
-
         symbolized_info = info.deep_transform_keys(&:to_sym)
 
         transformed_info = symbolized_info.reduce({}) do |acc, (k, v)|
-          if transform_targets.key? k
-            acc.merge({ transform_targets[k] => v })
+          if BENFICIARY_KEY_TRANFORMS.key? k
+            acc.merge({ BENFICIARY_KEY_TRANFORMS[k] => v })
```

Comment:
spelling, if I'm being picky :)

Reply:
Thank you for being picky! Totally overlooked that.